docs/schema.yaml

```diff
@@ -37,17 +38,23 @@ app_fdroid_latest:
 app_fdroid_version:
   id: str()
   version: str()
+  hash: str()
+  hash_type: str(required=False)
   repository: str(required=False)
   app_type: enum('system', 'privileged', 'user', required=False)
 
 app_fdroid_versioncode:
   id: str()
   versioncode: int()
+  hash: str()
+  hash_type: str(required=False)
   repository: str(required=False)
   app_type: enum('system', 'privileged', 'user', required=False)
 
 app_direct_download:
   id: str()
   name: str()
+  hash: str()
+  hash_type: str(required=False)
```

Comment:
I think that we should have the `hash` key required.

If one adds something from a remote location, one should have tested things in the first place...